WikiDrips/WikiRequest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  WikiRequest.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/3/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import Foundation
+import Accounts
+import Social
+import CoreLocation
+
+public class WikiRequest
+{
+//    func fetchWikiDoc() -> WikiDoc {
+//        return WikiDoc()
+//    }
+    
+    public func fetchWikiDoc(handler: ([WikiDoc]) -> Void) {
```

Comment:
`handler` should probably be an `@escaping` block since it will be called after method returns.